src/components/ChartContainer/D3IndicatorByLineChart.js

```diff
@@ -126,10 +208,16 @@ D3IndicatorByLineChart.propTypes = {
       date: PropTypes.string,
```

Comment:
The linter is also complaining here, kind of. Since you are using `byLineDisplayData[i].measure` online 170 it wants `measure` to be added in PropTypes